File: src/mispobjects/org.go
package mispobjects

type Org struct {
	Id string `json:"id"`
	Uuid string `json:"uuid"`
	Name string `json:"name"`
	Description string `json:"description,omitempty"`
	Type string `json:"type,omitempty"`
	Sector string `json:"sector,omitempty"`
	CreatedBy string `json:"created_by,omitempty"`
	CreatedByEmail string `json:"created_by_email,omitempty"`
	RestrictedToDomain []string `json:"restricted_to_domain,omitempty"`
	DateModified string `json:"date_modified,omitempty"`
	DateCreated string `json:"date_created,omitempty"`
	Nationality string `json:"nationality,omitempty"`
	UserCount string `json:"user_count"`
	Local bool `json:"local,omitempty"`
	Contacts string `json:"contacts,omitempty"`
	LandingPage string `json:"landing_page,omitempty"`
}

type OrgJsonNamed struct {
	Organisations Org `json:"Organisation"`
}

File: src/main/kotlin/com/devo/feeds/data/misp/Org.kt
package com.devo.feeds.data.misp

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Org(
    val id: String? = null,
    val uuid: String? = null,
    val name: String? = null,
    val description: String? = null,
    val type: String? = null,
    val sector: String? = null,
    @SerialName("created_by") val createdBy: String? = null,
    @SerialName("created_by_email") val createdByEmail: String? = null,
    @SerialName("restricted_to_domain") val restrictedToEmail: List<String>? = null,
    @SerialName("date_modified") val dateModified: String? = null,
    @SerialName("date_created") val dateCreated: String? = null,
    val nationality: String? = null,
    @SerialName("user_count") val userCount: String? = null,
    val local: Boolean? = null,
    val contacts: String? = null,
    @SerialName("landing_page") val landingPage: String? = null
)
